components/table/stories/table.stories.js

```diff
@@ -25,7 +26,7 @@ export default {
 				type: { summary: "string" },
 				category: "Component",
 			},
-			options: ["standard", "compact", "spacious"],
+			options: ["regular", "compact", "spacious"],
```

Comment:
"Regular" seemed to match our other density examples better, but if we want to keep it as standard I can revert this.